cmd/tpl/main.go

```diff
@@ -18,9 +18,9 @@ var version, commit, date string
 func main() {
 	log := logrus.New()
 	var input string
-	var vaultPrefix string
+	var keyPrefix string
```

Comment:
"key" is actually wrong (typo) i wanted to write just "prefix" and "mapping" since it now applys to both vault and azure and it seems a bit confusing to have it name "vaultPrefix" since it would imply it only applies to vault.

I am fine with leaving it as "vaultPrefix" and "vaultMapping"

Reply:
you can also write something like secretPrefix. key is a bit misleading imhp

Reply:
i will just change it back for now its schemantics and @zerok can change it to something that makes sense :D